frontend/src/components/navbar/AuditSidebar.tsx

```diff
@@ -53,153 +53,184 @@ type AuditSidebarProps = {
+            'font-semibold transition-opacity',
+            isCollapsed ? 'opacity-0 w-0' : 'opacity-100',
+          )}
+        >
+          {t('audit')}
+        </h2>
+        <button
+          aria-label={isCollapsed ? 'Expand sidebar' : 'Collapse sidebar'}
+          className="text-gray-400 hover:text-gray-100 hover:bg-gray-800 p-2 rounded-full transition-colors duration-200 z-10"
+          onClick={() => setIsCollapsed(!isCollapsed)}
+          type="button"
+        >
+          {isCollapsed ? (
+            <ChevronDown className="h-5 w-5" />
+          ) : (
+            <ChevronUp className="h-5 w-5" />
+          )}
+        </button>
+      </div>
+      <nav className="flex-1 overflow-y-auto py-4">
+        <ul className="space-y-2">
+          {menuItems.map(item => (
+            <li key={item.name}>
+              <Link
                 className={clsx(
-                  'flex-1 transition-opacity',
-                  isCollapsed && 'opacity-0 w-0 overflow-hidden',
+                  'w-full flex items-center justify-start gap-3 px-4 py-2 text-left text-sm font-medium rounded-lg transition-colors duration-200',
+                  'hover:bg-gray-800',
+                  activeItem === item.name
+                    ? 'bg-gray-800 text-white'
+                    : 'text-gray-400',
                 )}
+                onClick={() => {
+                  setActiveItem(item.name);
+                }}
+                to={item.value}
               >
-                {item.name}
-              </span>
-              {item.additionalIcon ? (
-                <item.additionalIcon
-                  className={clsx('h-4 w-4', isCollapsed ? 'ml-0' : 'ml-auto')}
-                />
-              ) : null}
-            </Link>
-          </li>
-        ))}
-      </ul>
-      <div className={clsx('mt-4 px-4', isCollapsed && 'px-2')}>
-        <div className={clsx('mb-2', isCollapsed && 'sr-only')}>
-          <SelectDropdown
-            items={sortOptions}
-            onChange={setSortBy}
-            placeholder={t('select')}
-            selected={sortBy}
-            title={t('sortBy')}
-          />
+                <item.icon className="h-5 w-5 flex-shrink-0" />
+                <span
+                  className={clsx(
+                    'flex-1 transition-opacity',
+                    isCollapsed && 'opacity-0 w-0 overflow-hidden',
+                  )}
+                >
+                  {item.name}
+                </span>
+                {item.additionalIcon ? (
+                  <item.additionalIcon
+                    className={clsx(
+                      'h-4 w-4',
+                      isCollapsed ? 'ml-0' : 'ml-auto',
+                    )}
+                  />
+                ) : null}
+              </Link>
+            </li>
+          ))}
+        </ul>
+        <div className={clsx('mt-4 px-4', isCollapsed && 'px-2')}>
+          <div className={clsx('mb-4 font-thin', isCollapsed && 'sr-only')}>
```

Comment:
```suggestion
          <div className={clsx('mb-4 font-light', isCollapsed && 'sr-only')}>
```

Este cambio mejora el contraste, ya que con font-thin no parece que la letra fuera de un color claro.